classes/functions.php

```diff
@@ -210,7 +210,7 @@ public function seriesList($listAllSeries) {
           $list .= "series_id=" . $row ['series_id'] . " or ";
         }
         $idList = preg_replace('/(or(?!.*or))/', '', $list);
-        $sql = "SELECT * FROM series where $idList ORDER BY series_name ASC";
+        $sql = "SELECT * FROM series where $idList ORDER BY series_name ASC, series_vol ASC";
```

Comment:
Sort by Series name and vol on the main index page. Now orders by volume number.